### PR TITLE
Enforce formatting through gradle

### PR DESCRIPTION
**Pull request by obj-obj**

### CODE REVIEW

**Comment by obj-obj** [comment]
Currently does not working when using `./gradlew test`, I will add that soon

**Comment by obj-obj** [comment]
Turns out `./gradlew check` works fine, idk what's adding `./gradlew test` but it's not in base gradle

**Comment by obj-obj** [comment]
@jamesdooley4 would this work?

**Comment by 1198159** [comment]
we dont need to enforce formatting we have freshmen for a reason

File: build.gradle [hunk 1]
```diff
@@ -16,10 +45,10 @@ targetCompatibility = JavaVersion.VERSION_11
 def ROBOT_MAIN_CLASS = "frc.team2412.robot.Main"
 
 // Define my targets (RoboRIO) and artifacts (deployable files)
-// This is added by GradleRIO's backing project DeployUtils.
+// This is added by GradleRIO"s backing project DeployUtils.
```
Comment (jamesdooley4) [inline build.gradle:48]: Please convert back to an apostrophe

**Review by jamesdooley4 — APPROVED** [review]

**Review by jamesdooley4 — APPROVED** [review]

**Pull request merged**